src/services/Sns.js

```diff
@@ -0,0 +1,37 @@
+const fetch = require('node-fetch');
+const cheerio = require('cheerio');
+
+const url = 'https://www.sns24.gov.pt/alertas/';
+
+/**
+ * Get alerts data form SNS24 website
+ *
+ * @returns {Array} data
+ */
+const getAlerts = async () => {
+  const data = await fetch(url)
+    .then(res => res.text())
+    .then((body) => {
+      const html = body;
```

Comment:
Why this intermediate variable?

Reply:
i assumed its necessary for html, https://github.com/bitinn/node-fetch#plain-text-or-html , but i will test.

Reply:
yes its needed, like in src/api/api.js.

Reply:
After a short discussion with @ClaudioVarandas we decided to remove the extra variable since we can rely on `body` to initialize `cheerio`.

```js
const $ = cheerio.load(body);
```